pkg/controller/constraint/constraint_controller.go

```diff
@@ -127,7 +127,7 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 		if err = util.SetHAStatus(instance, status); err != nil {
 			return reconcile.Result{}, err
 		}
-		if _, err := r.opa.AddConstraint(context.Background(), instance); err != nil {
+		if err := r.cacheConstraint(instance); err != nil {
```

Comment:
I don't think err needs to be instantiated here. linter should have caught this.

Reply:
Doesn't need to be but valid if it is. If instantiated it avoids clobbering whatever err was in the outer scope.

Does linter complain when you run it locally?

Reply:
No complaints when run locally for me.